### ISSUE TITLE
vcsam2p.ico hard link appears in sam2p-0.49.4.tar.gz

### ISSUE THREAD

**Issue by pts** [pts/sam2p#48]
Based on https://github.com/pts/sam2p/issues/47 reported by @dag-erling .

```
$ wget -qO sam2p-0.49.4.tar.gz https://github.com/pts/sam2p/releases/download/v0.49.4/sam2p-0.49.4.tar.gz
$ sha256sum sam2p-0.49.4.tar.gz
d23707b2405ca94e2a237cb81f62fb5916f5e4360cf9a705061479c8fa1dff5c  sam2p-0.49.4.tar.gz
$ tar tzf sam2p-0.49.4.tar.gz
(empty output)
$ tar xzf sam2p-0.49.4.tar.gz
(empty output)
$ tar tzvf sam2p-0.49.4.tar.gz | grep vcsam2p.ico 
-rw-r----- pts/eng        1078 2013-12-11 17:52 sam2p-0.49.4/vcsam2p.ico
hrw-r----- pts/eng           0 2013-12-11 17:52 sam2p-0.49.4/vcsam2p.ico link to sam2p-0.49.4/vcsam2p.ico
$ tar --version
tar (GNU tar) 1.30
(...)
```

The output line starting with `hrw` shouldn't appear.

Also `tar -cf-` should be used in `make dist` to make sure that `tar` writes to stdout (and not to `$TAPE` or some other compiled-in default).

**Comment by dag-erling** [comment]
It's not a symlink, it's a hard link. It seems that newer versions of GNU tar keep track of the device and inode number of every file they process and emit a hard link when they see a duplicate instead of including the same file twice. Attempting to link a file to itself when unpacking fails. #50 should fix this.

**Comment by pts** [comment]
Thank you for the fix! Fixed in 4c0ab7755a258aa5f3bc327b62a30ce80f41c63f.
